java/arcs/android/storage/service/IDevToolsStorageManager.aidl

```diff
@@ -22,4 +24,6 @@ interface IDevToolsStorageManager {
      */
     String getStorageKeys();
 
+    IDevToolsProxy getDevToolsProxy();
```

Comment:
Javadoc comments would be good here.

Also - `oneway` keyword will help a ton with performance.

Reply:
The oneway keyword doesn't work since getDevToolsProxy returns a value.

Reply:
Sorry, my bad!